### PR TITLE
My code review suggestions

### PR DESCRIPTION
**Pull request by bahnsm**
Project Description
This is an application for keeping track of medical appointments and prescription medication regimens. This program doesn't really do anything yet, though the concept is very promising. While there was not a lot of code to review, what there is so far is mostly neat, readable, and easy to understand. The program compiles and was available on GitHub on time.

Change List
-ExampleUnitTest: Changed name of addition_isCorrect to be more consistent with naming convention of other functions. Converted function to expression body.
-HomeActivity: Added spaces around mutableStateOf. Converted DefaultPreview2 to expression body. Converted call of ToDoItemFacts to single-line lambda. Moved first argument of each OutlinedTextField to new line.
-MainActivity: Converted call of Greeting to single-line lambda. Capitalized first word of comment above class declaration. Converted DefaultPreview to expression body. Converted Greeting to expression body.
-Appointment: Changed indentation to 4 spaces.
-Form: Changed indentation to 4 spaces.
-Medication: Changed indentation to 4 spaces.
-MetricTypes: Put parameters on 1 line.
-Notification: Changed indentation to 4 spaces.

Links for my commits to my own group's project
https://github.com/WillTaylor2352/Plan2Meet/commit/97833e3dd86956d3fa51306c0c821658bf76dcaa
https://github.com/WillTaylor2352/Plan2Meet/commit/9befdb6ebad3a8f24d43f7efe120b946281e5a4c
https://github.com/WillTaylor2352/Plan2Meet/commit/3effb1f7084b259e422b5c5b5d43f72244d77640

What I Learned
-I learned about function naming conventions in Kotlin: https://kotlinlang.org/docs/coding-conventions.html#naming-rules
-I learned to use expression bodies for functions containing a single expression: https://kotlinlang.org/docs/coding-conventions.html#functions
-I learned that comments are supposed to begin with a capital letter: https://kotlinlang.org/docs/coding-conventions.html#horizontal-whitespace
-I learned that indentation in Kotlin should be done with 4 spaces, NOT the Tab key: https://kotlinlang.org/docs/coding-conventions.html#indentation

### CODE REVIEW

File: app/src/main/java/com/medprompt/HomeActivity.kt [hunk 1]
```diff
@@ -27,9 +27,8 @@ public class HomeActivity : ComponentActivity() {
                 Surface(
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
-                ) {
-                    ToDoItemFacts("Android")
-                }
+                )
```
Comment (bahnsm) [inline app/src/main/java/com/medprompt/HomeActivity.kt:30]: Converted to single-line lambda
https://kotlinlang.org/docs/coding-conventions.html#lambdas

File: app/src/main/java/com/medprompt/HomeActivity.kt [hunk 2]
```diff
@@ -41,52 +40,53 @@ public class HomeActivity : ComponentActivity() {
...
+    var timeOfDay by remember { mutableStateOf("") }
+    var frequency by remember { mutableStateOf("") }
+    var doseSize by remember { mutableStateOf("") }
+    var stock by remember { mutableStateOf("") }
     val context = LocalContext.current
 
     Column {
-        OutlinedTextField(value = itemName,
+        OutlinedTextField(
+            value = itemName,
```
Comment (bahnsm) [inline app/src/main/java/com/medprompt/HomeActivity.kt:52]: Moved 'value' to new line so it aligns with other arguments
https://kotlinlang.org/docs/coding-conventions.html#class-headers

File: app/src/main/java/com/medprompt/HomeActivity.kt [hunk 3]
```diff
@@ -41,52 +40,53 @@ public class HomeActivity : ComponentActivity() {
...
+        Button(
             onClick = {
                 Toast.makeText(
                     context,
                     "$itemName $timeOfDay $frequency $doseSize $stock",
                     Toast.LENGTH_LONG
                 ).show()
             }
-        ){
+        ) {
```
Comment (bahnsm) [inline app/src/main/java/com/medprompt/HomeActivity.kt:84]: Added space to resolve warning

File: app/src/main/java/com/medprompt/HomeActivity.kt (hunk 1 above)
Comment (discospiff) [reply app/src/main/java/com/medprompt/HomeActivity.kt:30]: I'd keep this as it were.  There will likely be more lines added, where having the {} on a separate line is best.  It's also acceptable if there is only one line.

File: app/src/main/java/com/medprompt/HomeActivity.kt [hunk 4]
```diff
@@ -41,52 +40,53 @@ public class HomeActivity : ComponentActivity() {
...
 }
 
 @Preview(showBackground = true)
 @Composable
-fun DefaultPreview2() {
-    MedpromptTheme {
-        ToDoItemFacts(name = "")
-    }
-}
+fun DefaultPreview2() = MedpromptTheme { ToDoItemFacts(name = "") }
```
[the remark this answers is not in the document]
Comment (discospiff) [reply app/src/main/java/com/medprompt/HomeActivity.kt:92]: Either way works.  The previous way is less crowded; easier to add more lines, so I'd likely stick with that.

**Comment by discospiff** [comment]
Most of the changes are formatting.  Instead of doing this manually, use the IDE to do it for you, automatically.  That will save time, and be more standard.
Control - Alt - L Windows
Command - Alt - L Mac

No need to merge this branch, since these changes can be performed by the IDE.

**Pull request closed without merging**